projects/tensorflow/BUILD

```diff
@@ -0,0 +1,216 @@
+# Fuzzing TensorFlow.
```

Comment:
Can't we use this from the TF repo? We'll continue updating the file there as more fuzzers get written and we get internal/external in sync.

If we need to have this here, then maybe we should remove the python ones, as we have a separate project for that.

Reply:
Yes! The reason I put this here was simply for testing whether the fix worked or not and keeping track. The real goal was to simply get the CI to pass and then move onwards with updating the Tensorflow repo.

I have been to busy on other things to finish this, but if you would like to resume the fuzzing of Tensorflow then I am happy to put in the efforts to get it working and migrate changes upstream.